packages/chat-client/src/controllers/engine.ts

```diff
@@ -78,6 +78,19 @@ export class ChatEngine extends IChatEngine {
     return this.client.identityKeys.registerIdentity({ accountId, onSign });
   };
 
+  private deriveThreadResponseTopic = async (
+    pubkeyY: string,
+    pubkeyX: string
+  ) => {
+    const topicSymKeyI = await this.client.core.crypto.generateSharedKey(
```

Comment:
Why do we have a `I` suffix, can it be clearer ?

Reply:
Ah, it's to match the specs. I'd rather not change it, since whenever a spec change happens we talk about "pubkeyY" or "symkeyI"